CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v5.12.0 (unreleased)
+
+#### Notes
+Extended support to Enclosure for OneView Rest API version 1600.
```

Comment:
better not to specify resource name here since you have already mention it in the Features supported list.

Reply:
@soodpr updated

Reply:
I meant something like "This release extends support of the SDK to OneView REST API version 1600 (OneView v5.20)."

Reply:
@soodpr Updated.